ch_pipeline/hfb/analysis.py

```diff
@@ -1047,3 +1048,59 @@ def _ensure_list(x):
         y = [x]
 
     return y
+
+
+class HFBMedianSubtraction(task.SingleTask):
+    """Subtracting weighted median along beam axis to remove fluctuations in the
+    data induced by temperature fluctuations in East and West receiver huts."""
+
+    def process(self, stream):
+        """Subtract weighted median of the data along beam axis from the data.
+        A binary mask (0 when the weight is zero, and 1 for non-zero weights)
+        is used in weighted median
```

Comment:
There is a convention for docstrings that should be followed. They should be a one line short description (max 88 chars), and then additional details in a subsequent paragraph.